docs/install.rst

```diff
@@ -101,33 +101,6 @@ releases which are compatible with the latest ``pytest`` and ``sphinx`` releases
     Attempting to use `pip <https://pip.pypa.io>`__ to upgrade your installation
     of ``astropy`` itself may result in a corrupted installation.
 
-.. _testing_installed_astropy:
-
-Testing an Installed ``astropy``
-================================
-
-{% if is_development %}
-
-The easiest way to test if your installed version of ``astropy`` is running
-correctly is to use the :ref:`astropy.test()` function::
-
-    import astropy
-    astropy.test()
-
-The tests should run and print out any failures, which you can report at
-the `Astropy issue tracker <https://github.com/astropy/astropy/issues>`_.
-
-This way of running the tests may not work if you do it in the ``astropy`` source
-distribution. See :ref:`sourcebuildtest` for how to run the tests from the
-source code directory, or :ref:`running-tests` for more details.
-
-{%else%}
-
-See the :ref:`latest documentation on how to test your installed version of
-astropy <astropy-dev:testing_installed_astropy>`.
-
-{%endif%}
-
```

Comment:
We should keep this section and advertise `pytest --pyargs astropy` instead.

Reply:
This is for user facing install. I don't think we should really be asking a user to run `pytest` after doing a `pip`/`conda` install. It should Just Work™

The user would need to go look into the install paths to run pytest anyway and we would also need to explain how to first install pytest.

Reply:
We are not asking them to run tests, just documenting how to do it.

> It should Just Work™

And then there is real life...
This PR is about removing `astropy.test()`, not removing all tests or preventing users to run them on an installed package. And our release process still includes RC testing with the same method.

> we would also need to explain how to first install pytest.

Right, this is missing in this section. But `pip install pytest-astropy` is mentioned in the section just above about conda... this could be explained better, maybe just move the paragraph from the conda section to the test one (and remove sphinx-astropy, it is documented later).